### ISSUE TITLE
[vscode][project-references] broken links and missing completions

### ISSUE THREAD

**Issue by akosyakov** [microsoft/TypeScript#26467]
I am trying to migrate Theia to project references: https://github.com/theia-ide/theia/pull/2563/files

Besides that, the compilation does not work: https://github.com/Microsoft/TypeScript/issues/26460

Experience in VS Code is bogus sometimes, particularly there are broken links after rename refactoring and in completions I don't see symbols from referenced projects.

![code](https://user-images.githubusercontent.com/3082655/44138215-80fa0f40-a073-11e8-9a54-a30f56bc812c.gif)

**Comment by RyanCavanaugh** [comment]
I'm assuming you've unblocked yourself by now 😅 Please log a new issue if there are concrete repro steps to a bug.
